docs/concepts.md

```diff
@@ -159,24 +158,21 @@ following system diagram.
 
 ### Workflow
 
-In response to an application creation request, the Oak Manager sends back to
-the caller details about the gRPC endpoint of the newly created Oak Application,
-initialized with the application configuration specified in the request.
+ISV runs an Oak Application using the Oak Runner and publishes details about the
```

Comment:
Question for @daviddrysdale (as native english speaker): should this be "the ISV" or just "ISV" throughout?

Reply:
Probably "The ISV"

Reply:
Done
Also changed all insurances of a "Developer" to the "ISV"